graylog2-web-interface/flow-typed/npm/styled-components_v4.x.x.js

```diff
@@ -23,7 +23,10 @@ declare module 'styled-components' {
     | void
     | {[ruleOrSelector: string]: string | number, ...} // CSS-in-JS object returned by polished are also supported
 
-  declare export type TaggedTemplateLiteral<I, R> = (strings: string[], ...interpolations: Interpolation<I>[]) => R
+  declare export type TaggedTemplateLiteral<I, R> = {
```

Comment:
This was the only changed item - everything else is updated from an auto-running `eslint --fix`